chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -51,24 +51,15 @@
 default[:python][:installed] = "#{node[:python][:scripts]}\\#{node[:python][:pywin32register]}"
 
 default[:openstack][:location] = 'C:\OpenStack'
+default[:openstack][:tarball_branch] = 'stable-liberty'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)